products/plugins/commercial/index.md

```diff
@@ -22,3 +22,35 @@ This plugin contains various features, which are covered in our docs as well.
 ::: warning
 In accordance with a Shopware merchant's active account configuration, features within the plugin will be in *active* or *inactive* (whilst still being installed within the Shopware codebase). Pay close attention to any install information or special conditions for the provided features.
 :::
+
+## Licensing
+
+On installation the commercial plugin tries to fetch the license key using the logged-in Shopware Account. If this can't be fetched, the plugin can be still installed but all features are deactivated. If you login into your Shopware Account, you can fetch again the license key using `bin/console commercial:license:update`.
```

Comment:
```suggestion
On installation, the commercial plugin tries to fetch the license key using the logged-in Shopware Account. If this can't be fetched, the plugin can be installed, but all features are deactivated. If you log into your Shopware Account, you can fetch the license key again using `bin/console commercial:license:update`.
```